### PR TITLE
Csv data

### PR DESCRIPTION
**Pull request by charlesbaird-dsa**
Base catalogue file (apic.csv) broken out into 3 files -
operational.csv - dateadded/updated
catalogue.csv - data for catalogue
organisations.csv - list of orgs/abbreviations

also process_csv.rb, which adds uuids and breaks up the original file

### CODE REVIEW

File: data/operational.csv [hunk 1]
```diff
@@ -0,0 +1,212 @@
+dateAdded,dateUpdated,uuid
+08/01/2020,24/08/2020,ccfb7eaf-9dae-460f-9812-9e2fc0a31178
```
Comment (arnau) [inline data/operational.csv:2]: Fix dates to be ISO8601 that is `YYYY-MM-DD` e.g. "2020-08-01"

File: data/operational.csv [hunk 2]
```diff
@@ -0,0 +1,212 @@
+dateAdded,dateUpdated,uuid
```
Comment (arnau) [inline data/operational.csv:1]: The [schema](https://github.com/alphagov/api-catalogue/blob/master/data/operational.csv-metadata.json) calls the "uuid" column "about", and it is the first column. Also "dateCreated" and "dateModified".

File: data/organisations.csv [hunk 3]
```diff
@@ -0,0 +1,16 @@
+﻿organisation,abbrev
```
Comment (arnau) [inline data/organisations.csv:1]: The [schema](https://github.com/alphagov/api-catalogue/blob/master/data/organisation.csv-metadata.json) differs in column naming. Also the name of the file should be `organisation.csv` (i.e. in singular).

Comment (arnau) [reply data/organisations.csv:1]: Also, you need to add a synthetic identifier for every organisation as we can't rely on the name nor the acronym.

File: data/process_csv.rb [hunk 4]
```diff
@@ -0,0 +1,38 @@
+require 'csv'
```
Comment (arnau) [inline data/process_csv.rb:1]: What's the purpose of this script? Could you write a summary for it? When should be used?

File: source/dashboard/index.html.md.erb [hunk 5]
```diff
@@ -0,0 +1,33 @@
+---
+title: API Catalogue Contents
+weight: 80
+hide_in_navigation: true
+---
+
+# API Catalogue Contents
+
+<div style="height:1px;font-size:1px;">&nbsp;</div>
```
Comment (arnau) [inline source/dashboard/index.html.md.erb:9]: What is the intention of this?

File: source/dashboard/index.html.md.erb [hunk 6]
```diff
@@ -0,0 +1,33 @@
+---
```
Comment (arnau) [inline source/dashboard/index.html.md.erb:1]: Wasn't this in a different PR?

File: unpack_scripts/template.erb [hunk 7]
```diff
@@ -0,0 +1,29 @@
+---
```
Comment (arnau) [inline unpack_scripts/template.erb:1]: What is the purpose of this template?

File: unpack_scripts/unpack_apic.rb [hunk 8]
```diff
@@ -0,0 +1,79 @@
+require 'csv'
```
Comment (arnau) [inline unpack_scripts/unpack_apic.rb:1]: What's the purpose of this script?

**Review by arnau — CHANGES_REQUESTED** [review]
Could you:

* remove `data/apic.csv`
* remove `unpack_scripts/apic7.csv`
* remove `unpack_scripts/catalogue.csv`
* remove `unpack_scripts/db_test.rb` a test shouldn't be merged into master. If the script is used, rename it and add some documentation.
* the column names for `data/catalogue.csv` don't match the schema.

File: data/process_csv.rb (hunk 4 above)
Comment (charlesbaird-dsa) [reply data/process_csv.rb:1]: This is the script which takes apic.csv and splits it into two files, linked by UUID.

Comment (arnau) [reply data/process_csv.rb:1]: Could you write a summary in as a Ruby comment at the beginning of the file?

**Comment by charlesbaird-dsa** [comment]
Organisation and catalogue.csv now updated

**Comment by odlp** [comment]
@charlesbaird-dsa can we close this PR now? Perhaps we can keep the branch if we need to keep a reference

**Pull request closed without merging**